bigquery/samples/client_query.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+def client_query(client):
+
+    # [START bigquery_query]
+    # TODO(developer): Import the client library.
+    # from google.cloud import bigquery
+
+    # TODO(developer): Construct a BigQuery client object.
+    # client = bigquery.Client()
+
+    query = (
+        "SELECT name FROM `bigquery-public-data.usa_names.usa_1910_2013` "
+        'WHERE state = "TX" '
+        "LIMIT 100"
+    )
+    query_job = client.query(
+        query, location="US"  # Must match the destination dataset(s) location.
+    )  # Make an API request.
+
+    print("The query data:")
+    for row in query_job:
+        # Row values can be accessed by field name or index
+        print(row)
```

Comment:
Let's demonstrate that fields can be accessed by name or index.

```suggestion
        print("name={}, count={}".format(row[0], row["count"]))
```